src/c_api/c_api_profile.cc

```diff
@@ -314,8 +323,15 @@ int MXAggregateProfileStatsPrint(const char **out_str, int reset) {
     std::shared_ptr<profiler::AggregateStats> stats = profiler->GetAggregateStats();
     std::ostringstream os;
     if (stats) {
-      stats->Dump(os, reset != 0);
+      if (static_cast<PrintFormat>(format) == PrintFormat::table)
+        stats->DumpTable(os, sort_by, ascending);
+      else if (static_cast<PrintFormat>(format) == PrintFormat::json)
+        stats->DumpJson(os, sort_by, ascending);
+      else
+        LOG(FATAL) << "Invliad value for parameter format";
```

Comment:
Let us make it more actionable message for user. Something like - "Unknown format provided - " << format << "Supported formats - ' json', 'table'" ?

Same for sort parameter

Reply:
This is in backend so format and sort_by are int. I will add the warning you described to frontend

Reply:
Nit: spell issue